system_baseline/views/v0.py

```diff
@@ -131,9 +157,27 @@ def delete_baselines_by_ids(baseline_ids):
     return "OK"
 
 
+def _create_ordering(order_by, order_how, query):
+    """
+    helper method to set ordering on query
+    """
+    if order_by == "display_name":
+        if order_how == "ASC":
+            query = query.order_by(SystemBaseline.display_name.asc())
+        else:
```

Comment:
Does something gate the value of `order_how` ? The naked else implies no matter what I pass for `order_how` I will get a descending sort.

Reply:
its gated to either DESC or ASC in the openapi yaml. However I rewrote it a bit to make it clearer elsewhere, will update here.